src/lib/rest/RestService.cpp

```diff
@@ -759,7 +759,7 @@ static std::string restService(ConnectionInfo* ciP, RestService* serviceV)
   std::string  details = std::string("service '") + ciP->url + "' not recognized";
   std::string  answer;
 
-  restErrorReplyGet(ciP, SccBadRequest, "service not found", &answer);
+  restErrorReplyGet(ciP, SccBadRequest, "Service not found. Check your URL as probably it is wrong.", &answer);
```

Comment:
Note you are repeating four times the same error message. I thinks it would be better to define it only one time in errorMessages.h and use that definition.

In particular:

```
#define ERROR_DESC_BAD_REQUEST_SERVICE_NOT_FOUND   "Service not found. Check your URL as probably it is wrong."
```